packages/components/src/Breadcrumbs/Breadcrumbs.component.js

```diff
@@ -109,15 +122,20 @@ function Breadcrumbs(props) {
 	}
 
 	return (
-		<ol id={props.id} className={classNames('breadcrumb', theme['tc-breadcrumb'], 'tc-breadcrumb')}>
+		<ol
+			id={props.id}
+			className={classNames('breadcrumb', theme['tc-breadcrumb'], 'tc-breadcrumb')}
+			role="navigation"
```

Comment:
⚠️ Be careful with [that](https://github.com/Talend/ui/blob/master/packages/theme/src/theme/variations/_tdp.scss#L12)

Reply:
Oh thank you, I'll remove it on Breadcrumb

Reply:
Here is a good [resource](https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/breadcrumb/index.html)